src/index.c

```diff
@@ -1606,7 +1606,7 @@ PRINT_PROFILE_FUNC(printUnionIt) {
   // if MAXPREFIXEXPANSIONS reached
   if (ui->norig == config->iteratorsConfig->maxPrefixExpansions) {
     RedisModule_Reply_SimpleString(reply, "Warning");
-    RedisModule_Reply_SimpleString(reply, "Max prefix expansion reached");
+    RedisModule_Reply_SimpleString(reply, "Max prefix expansions limit was reached");
```

Comment:
Agreeing that rephrasing the error message is not considered a breaking change since it is in `FT.PROFILE`